bundle/metadata/dependencies.yaml

```diff
@@ -15,3 +15,7 @@ dependencies:
   value:
     packageName: csi-addons
     version: "0.7.0"
+- type: olm.package
+  value:
+    packageName: ose-prometheus-operator
```

Comment:
Why are we setting the package names as `ose-prometheus-operator` and not `odf-prometheus-operator`? Any specific reason?

Reply:
It was a mistake. Have updated